irohad/network/impl/block_loader_service.cpp

```diff
@@ -58,23 +58,24 @@ grpc::Status BlockLoaderService::retrieveBlock(
 
   // try to fetch block from the consensus cache
   auto block = consensus_result_cache_->get();
-  if (block and block->hash() == hash) {
-    auto transport_block =
-        std::static_pointer_cast<shared_model::proto::Block>(block)
-            ->getTransport();
-    response->CopyFrom(transport_block);
-    return grpc::Status::OK;
-  } else if (not block) {
-    log_->info(
-        "Tried to retrieve a block from an empty cache: requested block hash "
-        "{}",
-        hash.hex());
-  } else {
+  if (block) {
+    if (block->hash() == hash) {
+      auto transport_block =
+          std::static_pointer_cast<shared_model::proto::Block>(block)
+              ->getTransport();
+      response->CopyFrom(transport_block);
+      return grpc::Status::OK;
+    }
     log_->info(
```

Comment:
Looks like missed else branch here.

Reply:
It should be decided whether we want explicit else branches if there is control termination in if branch.

Reply:
Yeah, else is not necessary here, I am for implicit `else`, as there is `return` one line before, and this reduces nesting

Reply:
Better to make it explicit, if possible.